src/constants/tokenSafety.tsx

```diff
@@ -103,3 +109,7 @@ export function checkSearchTokenWarning(token: SearchToken) {
   }
   return checkWarning(token.address)
 }
+
+export function displayWarningLabel(warning: Warning | null) {
```

Comment:
i don't follow. why would we only not show medium?

Reply:
From my understanding medium warnings are only used for tokens on the ExtendedToken list which wanted to be made less as a "warning" and more of a message.

I just followed the designs on this Figma. https://www.figma.com/file/ZBwsYUDt154qGsMegYXrIN/Token-Safety-Labels-v2?node-id=0%3A1&t=lrLiiRbxNmfpaGDn-1

Reply:
oh okay i think i follow